webpack.config.js

```diff
@@ -92,4 +92,7 @@ module.exports = {
     },
   },
   bail: true,
+  node: {
+    fs: "empty",
+  },
```

Comment:
what's this?

Reply:
my bad, should have removed this, previously was adding twin macro and needed this for smt, but dont need it anymore.